src/python/grpcio_health_checking/MANIFEST.in

```diff
@@ -1,4 +1,4 @@
 include grpc_version.py
 include health_commands.py
-graft grpc
```

Comment:
It's changed for the same reason as everything else: things used to be in a package called `grpc`, and are now in a package called `grpc_$SOMETHING`.